<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>habits_module</class>
 <widget class="QWidget" name="habits_module">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: white;
    padding: 10px;
	margin: 60px 20px 80px;
	margin: 0px auto;
}

QLabel {
  color: black;
}

QCheckBox {
  color: black;
}

QPushButton {
	color: black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="habit_title_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_title">
        <property name="styleSheet">
         <string notr="true">QLabel {
	font-weight: bold;
	font-size: 20px;
}
</string>
        </property>
        <property name="text">
         <string>Nuevo hábito</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="name_widget" native="true">
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_name_title">
        <property name="styleSheet">
         <string notr="true">QLabel {
  font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 2px solid lightgray;
	border-radius: 5px;
}</string>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="desc_widget" native="true">
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit {
	color: black;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_desc_title">
        <property name="styleSheet">
         <string notr="true">QLabel {
  font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>Descripción</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainText_desc">
        <property name="styleSheet">
         <string notr="true">QPlainTextEdit {
	border: 2px solid lightgray;
	border-radius: 5px;
}</string>
        </property>
        <property name="plainText">
         <string>Descripción</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="frequency_widget" native="true">
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    border: 2px solid lightgray;
    border-radius: 3px;
}

QCheckBox::indicator:checked {
	background-color: #3498db;
}

QWidget {
  padding: 0;
  margin-top: 0;
  margin-bottom: 0;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string>Jueves</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBox_9">
        <property name="text">
         <string>Domingo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_freq_title">
        <property name="styleSheet">
         <string notr="true">QLabel {
  font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>Frecuencia</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_10">
        <property name="text">
         <string>Miércoles</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_11">
        <property name="text">
         <string>Viernes</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_12">
        <property name="text">
         <string>Sábado</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_13">
        <property name="text">
         <string>Martes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_14">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Lunes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttons_widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
	margin: 0px 70px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #3498db;
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_message">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Puedes editar o eliminar este hábito luego en la sección de Detalle</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
